bleak/__init__.py

```diff
@@ -12,7 +12,6 @@
 import asyncio
 
 from bleak.__version__ import __version__  # noqa: F401
-from bleak.backends.bluezdbus import check_bluez_version
 from bleak.exc import BleakError
 
 _on_rtd = os.environ.get("READTHEDOCS") == "True"
```

Comment:
We can remove the `_on_ci = "CI" in os.environ` line below this line too since it is no longer used.

Reply:
Re-squashed and pushed with this removed